environment/providers/configmap/configmap.go

```diff
@@ -0,0 +1,21 @@
+package configmap
```

Comment:
Please add a `TODO` so we can keep track of the fact that configmap provider needs to be implemented.